### PR TITLE
fixes ColorComponent admin display and tweaks styling

### PR DESCRIPTION
**Pull request by mdole**
Follow-up from [377](https://github.com/artsy/palette/pull/377)

### CODE REVIEW

**Comment by damassi** [comment]
@mdole - mind posting a screen cap of the colors page?

**Comment by mdole** [comment]
of course! @damassi 
screencapture-localhost-8000-tokens-color-2019-04-22-16_14_58

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit f72037939838e0972c9917ed3451024214c6169c

https://deploy-preview-390--artsy-palette.netlify.com

File: packages/palette-docs/content/docs/tokens/Color.mdx [hunk 1]
```diff
@@ -2,74 +2,74 @@
...
+## Primary colors
 
 <ColorComponent color="black100" />
 
-This is our primary brand color. It is used as the default text color on light backgrounds and as the primary color of our logo. 
+This is our primary brand color. It is used as the default text color on light
+backgrounds and as the primary color of our logo.
 
-- - -
+<Spacer m={6} />
```
Comment (damassi) [inline packages/palette-docs/content/docs/tokens/Color.mdx]: Maybe `<Spacer my={4} />`? Maybe a littleee less whitespace.

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit 0ca0d8d3d963a9c36022e733e6b39e3bb4d55187

https://deploy-preview-390--artsy-palette.netlify.com

File: packages/palette-docs/content/docs/tokens/Color.mdx (hunk 1 above)
Comment (mdole) [reply packages/palette-docs/content/docs/tokens/Color.mdx]: :+1: 
screencapture-localhost-8000-tokens-color-2019-04-22-16_45_13

Comment (damassi) [reply packages/palette-docs/content/docs/tokens/Color.mdx]: There we go 👍

**Comment by artsyit** [comment]
:rocket: PR was released in v4.9.5 :rocket:

**Pull request merged**